Public/Src/Utilities/Configuration/ILoggingConfiguration.cs

```diff
@@ -118,7 +118,7 @@ public interface ILoggingConfiguration : IWarningHandling
         /// Creates a custom log file for a specific set of event IDs. Event list should be comma separated integers excluding the DX prefix.

```

Comment:
>Event list should be comma separated integers excluding the DX prefix. [](start = 71, length = 70)

This doc looks like its stale from CustomLog being a string. Could you update and include what EventLevel should be set to & what it means if unspecified?